app/views/travels/_form.html.erb

```diff
@@ -1,57 +1,187 @@
-<%= form_with(model: travel, local: true) do |form| %>
-  <% if travel.errors.any? %>
+<div>
+<%= form_with(model: [@current_enrollment, @travel], local: true) do |form| %>
+  <% if @travel.errors.any? %>
     <div id="error_explanation">
-      <h2><%= pluralize(travel.errors.count, "error") %> prohibited this travel from being saved:</h2>
-
+      <h2><%= pluralize(@travel.errors.count, "error") %> prohibited this travel from being saved:</h2>
       <ul>
-        <% travel.errors.full_messages.each do |message| %>
+        <% @travel.errors.full_messages.each do |message| %>
           <li><%= message %></li>
         <% end %>
       </ul>
     </div>
   <% end %>
+  <div class="text-sm" data-controller="travel">
+    <span>
+      PLEASE NOTE:  If you are no longer in need of the shuttle service you may have selected in your MMSS application, 
+      or if you have any questions about shuttle service, please email mmss@umich.edu as soon as possible.
+    </span>
+    <div class="py-2">
+      All fields are required unless marked <span class="text-gray-700">(optional)</span>
+    </div>
+    <hr>
+    <%= form.label 'travel[arrival_session]', 'Session of Arrival *' %>
+    <div class="form-check">
+      <% @sessions.each_with_index do |s, index| %>
+        <div>
+          <% if index == 0 %>
+            <input class="form-check-input appearance-none rounded-full h-4 w-4 border border-gray-600 bg-white checked:bg-blue-600 checked:border-blue-600 focus:outline-none transition duration-200 mt-1 align-top bg-no-repeat bg-center bg-contain float-left mr-2 cursor-pointer" 
+            type="radio" name="travel[arrival_session]" id="<%= s %>" value="<%= s %>" 
```

Comment:
I believe there is a way to deal with the radio buttons that is a Rails idiom

Reply:
I replaced radio buttons with drop-down lists